modules/@apostrophecms/notification/ui/apos/components/AposNotification.vue

```diff
@@ -127,7 +133,16 @@ export default {
         console.error('Unable to find notification job:', this.notification.jobId);
         this.job = null;
       }
-
+    }
+    // Notifications may include events to emit.
+    if (this.notification.event?.name) {
+      // Clear the event to make sure it's only emitted once across browsers.
+      const result = await this.clearEvent(this.notification._id);
+      // The result includes the number of documents found with the event.
+      // The notification doc will only still have the event in one instance.
+      if (result.n) {
```

Comment:
This is correct, but it's low-level mongo stuff, and there's no guarantee mongo won't reveal things we'd rather it didn't. It would be more appropriate to use an HTTP status code other than 200 in the case where it's already been cleared, or if there's no good one, to use a result property specific to this API rather than passing raw results from the mongo driver to the browser.